### ISSUE TITLE
MyAvatarEditor doesn't respond to changes in the editor

### ISSUE THREAD

**Issue by DraconicNEO** [ruffle-rs/ruffle#19448]
### Describe the bug

When I run the app, I can click on the buttons or change the sliders in the different options of the editor, but none of them do anything or have any effect on the program or the avatar display. The console generates a lot of errors too.

### Expected behavior

Clicking the different button options or moving the sliders should change the Miis and shouldn't give errors in the dev console. 

### Content Location

https://rc24.xyz/goodies/mii/

### Affected platform

Browser's extension

### Operating system

SteamOS 3.6.20

### Browser

Mozilla Firefox 134.0.2

### Additional information

Dev Console log:
```
Uncaught TypeError: addStylesNode is null
    loadDeferredStyles https://rc24.xyz/goodies/mii/:70
mii:70:7
Cookie warnings 5
using deprecated parameters for the initialization function; pass a single object instead 69.js:4120:21
INFO web/src/lib.rs:1251 Ruffle WASM module has been initialized 69.js:2589:21
WEBGL_debug_renderer_info is deprecated in Firefox and will be removed. Please use RENDERER. 69.js:2046:26
An AudioContext was prevented from starting automatically. It must be created or resumed after a user gesture on the page. 69.js:2870:21
An AudioContext was prevented from starting automatically. It must be created or resumed after a user gesture on the page. 2 69.js:3519:14
New Ruffle instance created (Version: nightly 2025-01-11 | WebAssembly extensions: ON | Used renderer: wgpu-webgl) ruffle.js:4659:17
Loading SWF file myavatareditor.swf ruffle.js:4818:25
ERROR core/src/avm2/events.rs:429 Error dispatching event EventObject(EventObject { type: "init", class: flash.events::Event, ptr: 0x16d2798 }) to handler FunctionObject(FunctionObject { ptr: 0x1306c30, name: Ok("com.myavatareditor.fla.editor::CharacteristicsLoader/graphicsLoaded()") }) : RustError("Unknown op") 69.js:2589:21
ERROR core/src/avm2/events.rs:429 Error dispatching event EventObject(EventObject { type: "click", class: flash.events::MouseEvent, ptr: 0x1334780 }) to handler FunctionObject(FunctionObject { ptr: 0x17233e8, name: Ok("com.myavatareditor.fla.editor::AbstractCharacteristicsGUI/selectColorHandler()") }) : RustError("Unknown op") 69.js:2589:21
ERROR core/src/avm2/events.rs:429 Error dispatching event EventObject(EventObject { type: "click", class: flash.events::MouseEvent, ptr: 0x14825e0 }) to handler FunctionObject(FunctionObject { ptr: 0x17233e8, name: Ok("com.myavatareditor.fla.editor::AbstractCharacteristicsGUI/selectColorHandler()") }) : RustError("Unknown op") 69.js:2589:21
ERROR core/src/avm2/events.rs:429 Error dispatching event EventObject(EventObject { type: "click", class: flash.events::MouseEvent, ptr: 0xfec220 }) to handler FunctionObject(FunctionObject { ptr: 0x17233e8, name: Ok("com.myavatareditor.fla.editor::AbstractCharacteristicsGUI/selectColorHandler()") }) : RustError("Unknown op") 69.js:2589:21
ERROR core/src/avm2/events.rs:429 Error dispatching event EventObject(EventObject { type: "click", class: flash.events::MouseEvent, ptr: 0x1351058 }) to handler FunctionObject(FunctionObject { ptr: 0x17233e8, name: Ok("com.myavatareditor.fla.editor::AbstractCharacteristicsGUI/selectColorHandler()") }) : RustError("Unknown op") 69.js:2589:21
ERROR core/src/avm2/events.rs:429 Error dispatching event EventObject(EventObject { type: "click", class: flash.events::MouseEvent, ptr: 0x14f3928 }) to handler FunctionObject(FunctionObject { ptr: 0x17233e8, name: Ok("com.myavatareditor.fla.editor::AbstractCharacteristicsGUI/selectColorHandler()") }) : RustError("Unknown op") 69.js:2589:21
ERROR core/src/avm2/events.rs:429 Error dispatching event EventObject(EventObject { type: "click", class: flash.events::MouseEvent, ptr: 0x14d1e18 }) to handler FunctionObject(FunctionObject { ptr: 0x17233e8, name: Ok("com.myavatareditor.fla.editor::AbstractCharacteristicsGUI/selectColorHandler()") }) : RustError("Unknown op") 69.js:2589:21

```
